packages/cubejs-server-core/core/index.js

```diff
@@ -41,6 +41,29 @@ const checkEnvForPlaceholders = () => {
   }
 };
 
+const devLogger = (type, message, error) => {
+  const colors = {
+    red: '31', // ERROR
+    green: '32', // INFO
+    blue: '34', // TRACE
+  }
+  const withColor = (str, color = colors.green) => `\u001b[${color}m${str}\u001b[0m`;
+  const format = json => JSON.stringify(json, null, 2);
+
+  if ([ // INFO list
```

Comment:
Can we do it configurable? Using some env like `CUBEJS_LOG_LEVEL=TRACE`?

Reply:
It looks like I misunderstand your words, log level should depend on env config rather than event itself.

Reply:
Yep. Right.